scripts/pylib/twister/harness.py

```diff
@@ -257,6 +265,10 @@ def handle(self, line):
             if tc.status == "skipped":
                 tc.reason = "ztest skip"
             tc.duration = float(match.group(4))
+            if tc.status in ["error", "failed", "timeout"]:
```

Comment:
the matched testcase status from output by this "re.compile(".*(PASS|FAIL|SKIP) - (test_)?" only have three status: passed, failed and skipped, for other two status error and timeout, they were due to testcase running blocked, we can't catch this line "re.compile(".*(PASS|FAIL|SKIP) - (test_)?", we can't handle it here, maybe we can fill it in each handler later.